SPECS-EXTENDED/oprofile/oprofile.spec

```diff
@@ -1,7 +1,5 @@
-%global java_home %(find %{_libdir}/jvm -name "OpenJDK*")
-
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
+%global java_home %(find %{_libdir}/jvm -name "msopenjdk*")
```

Comment:
Actually, right now it should be enough to add a BR on `javapackages-local-bootstrap` and remove local definitions for `java_home` completely. [I've been updating some files](https://github.com/microsoft/CBL-Mariner/pull/1893/files) a while ago and tried to do a clean-up of all usages of `java_home` - looks like I missed a spot.

Reply:
Thanks for the tip! Just pushed a commit to remove this line and use `javapackages-local-bootstrap` to pull in the macro and java package.